Wrappers/Python/ccpi/viewer/utils/conversion.py

```diff
@@ -1189,7 +1191,13 @@ def RequestData(self, request, inInfo, outInfo):
                 reader.Modified()
                 reader.Update()
                 # print(reader.GetOutput().GetScalarComponentAsDouble(0, 0, 0, 0))
-                outData.ShallowCopy(reader.GetOutput())
+                data = reader.GetOutput()
+                if self.__class__.__name__ == "cilTIFFResampleReader" and VTK_MAJOR_VERSION == 8:
+                    # In this case we have to flip along the y axis
```

Comment:
Why is that? Is there any VTK change that we can quote? Or is it just a workaround?

Reply:
See https://github.com/vais-ral/CILViewer/issues/296#issuecomment-1199037554

Reply:
Sorry, I posted the wrong link, meant to post this one https://github.com/vais-ral/CILViewer/issues/296#issuecomment-1198007102